docs/content/ingestion/native_tasks.md

```diff
@@ -0,0 +1,523 @@
+---
+layout: doc_page
+---
+# Native Index Tasks
+
+Druid currently has two types of native index tasks, i.e., `local index task` and `single phase parallel index task`.
+
+Single Phase Parallel Index Task
+--------------------------------
+
+The Single Phase Parallel Index Task is a task for parallel batch indexing. This task only uses Druid's resource and
+doesn't depend on other external systems like Hadoop. This task works in a single phase without shuffling intermediate
+data. `index_single_phase_parallel` task is a supervisor task which basically generates multiple worker tasks and submits
+them to overlords. Each worker task reads input data and makes segments. Once they successfully generate segments for all
+input, they report the generated segment list to the supervisor task. The supervisor task periodically checks the worker
+task statuses. If one of them fails, it retries the failed task until the retrying number reaches the configured limit.
+If all worker tasks succeed, then it collects the reported list of generated segments and publishes those segments at once.
+
+To use this task, the `firehose` in `ioConfig` should be _splittable_. If it's not, this task runs sequentially. The
```

Comment:
I think it would be worth elaborating on the mechanics of these splits, i.e. task per file blob or whatever, either here or in the docs for the splittable firehoses themselves.

Reply:
Good point. Also I fixed the broken links to the firehoses.